site/app/templates/polls/PollPreview.twig

```diff
@@ -0,0 +1,31 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}poll-preview{% endblock %}
+{% block title %}Student view of the poll:{% endblock %}
+{% block body %}
+<div>
+    <div class="content">
+        <h1 id="poll-preview-name">Poll Name</h1>
+        <div>
+            <p id="poll-preview-question"></p>
```

Comment:
This does not support mark down like normal polls do. This is one of the benefits to switching to the normal student twig templates.